### ISSUE TITLE
Background / Foreground on CustomChoices not working

### ISSUE THREAD

**Issue by coderazzi** [coderazzi/tablefilter-swing#25]
(Original issue 25 created by None on 2012-09-02T12:16:15+00:00)

```
What steps will reproduce the problem?
1. Create a CustomChoice class that defines a background and/or foreground
2. Add it to a FilterEditor

What is the expected output? What do you see instead?

The CustomChoice element in the Popup menu should appear in the provided color. That
does not happen

What version of the product are you using? On what operating system?

4.4.0 on Windows XP

Please provide any additional information below.


```

Original issue reported on code.google.com by `luismpena` on 2012-09-02 12:16:15

**Comment by coderazzi** [comment]
(Original issue 25 last updated on 2012-09-02T12:16:15+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by coderazzi** [comment]
(Original comment by coderazzi on 2012-09-02T14:05:51+00:00)

```
Solved on version 4.4.1
```

Original issue reported on code.google.com by `coderazzi` on 2012-09-02 14:05:51
